### PR TITLE
Update post

### PR DESCRIPTION
**Pull request by saracope**
This PR implements the following **changes:**

* update post to include search results display info

**URL / Link to page**
https://digital.gov/2020/05/11/experimenting-with-specialannouncement-markup

### CODE REVIEW

**Comment by saracope** [comment]
@afeijoo I need a larger size for the second image (rich-results-test). A width over 600 would do it I think.

**Comment by afeijoo** [comment]
Looks good other than the too-small `rich-results-test` image.

File: content/news/2020/05/2020-05-11-experimenting-with-specialannouncement-markup.md [hunk 1]
```diff
@@ -114,27 +118,26 @@ If the markup can’t be managed holistically, the expiration date field can be
...
+Search engines provide validation tools for structured data. These tools allow you to submit a url for a web page, and then test to see if the code is correct for the types of structured data that the search engine supports. If there’s a problem, they can give you some clues on how to fix it. The Bing tool requires a [Bing Webmaster Tools](https://www.bing.com/toolbox/webmaster) account. The [Google rich results tester](https://search.google.com/test/rich-results) does not have such a requirement. Both tools know to look for and check SpecialAnnouncement. Google’s rich results tester now provides a preview of the rich result too. The style does not match what we are seeing in actual search results, but the structure, interactivity, and text are the same.  
+
+{{< img-right src="rich-results-test" >}}
```
Comment (afeijoo) [inline content/news/2020/05/2020-05-11-experimenting-with-specialannouncement-markup.md:123]: Delete lines 123-125 to remove the image and the reference to it?

**Comment by saracope** [comment]
Updated to reflect the new publish date of June 9th and updated the image based on convo in Slack
@ToniBonittoGSA @afeijoo

**Pull request merged**